addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -99,6 +99,9 @@ export class Many2ManyTagsField extends Component {
             if (!recordlist) {
                 return;
             }
+            recordlist = recordlist.filter((element) => {
```

Comment:
besides the removal of the domain (diff in getDomain), this is the only other diff that should be there, I think (i.e. you don't filter out records that are already in the relation, and when the user clicks on them, you make sur we don't add them a second time to the relation). The remaining diff shouldn't exist (or I'm missing something)

Reply:
Also, the new behavior should be tested (I'm surprised no tests fail, actually)